ui/src/views/Tasks/CreateTask/taskList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
```

Comment:
No need to create a new component. Bring this logic to `src/views/Tasks/CreateTask/index.jsx`.